web/app/js/components/MetricsTable.jsx

```diff
@@ -59,6 +59,15 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
     }
   ];
 
+  let meshedColumn = {
```

Comment:
note that meshedColumn doesn't depend on any of the arguments to `columnDefinitions`, so it can be moved out into a separate const.